frontend/src/modules/app/ToolBar.tsx

```diff
@@ -75,7 +75,34 @@ const useStyles = makeStyles((theme: Theme) => ({
  */
 export const HeaderClock = (): JSX.Element => {
   const classes = useStyles();
-  const clockTime = useSelector(getClockTime);
+  // we can initialize with something more specific like wall clock time
+  const [clockTime, setClockTime] = useState(
+    new Date().toLocaleTimeString([], {
+      hour: '2-digit',
+      minute: '2-digit',
+      second: '2-digit',
+      hour12: true,
+    }),
+  );
+
+  /**
+   * UseEffect to run the clock every second
+   */
+  useEffect(() => {
+    const clockTimer = setInterval(() => {
+      setClockTime(
+        new Date().toLocaleTimeString([], {
+          hour: '2-digit',
+          minute: '2-digit',
+          second: '2-digit',
+          hour12: true,
+        }),
+      );
+    }, 1000);
```

Comment:
If performance stays the same, the wall clock display will be more synchronized with actual wall clock time (incl. when the seconds digits change) if we use an interval of 500, 250, or 100 Ms.

Reply:
oh right! i will change that